Driverless AI/ES/introducción-a-machine-learning-automático-con-driverless-ai/introducción-a-machine-learning-automático-con-driverless-ai.md

```diff
@@ -730,105 +730,97 @@ Aprende más sobre el gráfico de K-S en [Tutorial de análisis y puntuación de
-
+ 
 2\. Una vez que el modelo MLI esté completo, se debería de ver una imagen similar a la siguiente:
-
+ 
 ![finishing-up-mli](assets/finishing-up-mli.jpg)
-
+ 
 3\. Una vez que el **Experimento de MLI** haya terminado, aparecerá una ventana emergente, vaya a la página de MLI haciendo clic en **YES**.
-
+ 
 4\. La página de interpretabilidad de MLI tiene las explicaciones de los resultados del modelo en un formato legible para humanos:
-
+ 
 Esta sección describe la funcionalidad y características de MLI para experimentos regulares. Para experimentos que no son series de tiempo, esta página proporciona varias explicaciones visuales y códigos de razón para el modelo de Driverless AI entrenado, y sus resultados.
-
+ 
 ![mli-report-page-1](assets/mli-report-page-1.jpg)
-
-
+ 
 ![mli-report-page-2](assets/mli-report-page-2.jpg)
-
+ 
 *Cosas a tener en cuenta:*
-
+ 
 1. Resumen: resumen del experimento MLI. Esta página proporciona una visión general de la interpretación, además del conjunto de datos y el experimento de Driverless AI (si está disponible) que se utilizaron para la interpretación junto con el espacio de características (original o transformado), columna objetivo, tipo de problema e información de k-Lime.
-
+ 
 2. Modelo de Driverless AI: Para los experimentos de regresión y clasificación binaria, el menú Modelo de Driverless AI proporciona los siguientes gráficos para los modelos de Driverless AI:
-
+ 
     - **Feature Importance for transformed features:**(Importancia de funciones para funciones transformadas:): Este gráfico muestra la importancia de la función Driverless AI. La importancia de la característica de Driverless AI es una medida de la contribución de una variable de entrada a las predicciones generales del modelo de Driverless AI. La importancia de la característica global se calcula agregando la mejora en el criterio de división causada por una sola variable en todos los árboles de decisión en el modelo de Driverless AI.
-
+ 
   ![dai-model-feature-importance](assets/dai-model-feature-importance.jpg)
-
+ 
 - **Shapley plots for transformed features:**(Gráficos de Shapley para características transformadas:) para características transformadas: Las explicaciones de Shapley son una técnica con apoyo teórico creíble que presenta contribuciones variables globales y locales consistentes. Los valores numéricos locales de Shapley se calculan rastreando filas individuales de datos a través de un conjunto de árbol entrenado y agregando la contribución de cada variable de entrada a medida que la fila de datos se mueve a través del conjunto entrenado. Para las tareas de regresión, los valores de Shapley suman la predicción del modelo de IA sin conductor. Para problemas de clasificación, los valores de Shapley suman la predicción del modelo de Driverless AI antes de aplicar la función de enlace. Los valores globales de Shapley son el promedio de los valores absolutos de Shapley en cada fila de un conjunto de datos.
```

Comment:
Please check punctuation ":" in this sentence: 
Shapley plots for transformed features**:**(Gráficos de Shapley para características transformadas**:**) para características transformadas**:**